EphraimDev/way-farer
# #166968465 Add user signup endpoint

#### What does this PR do?
Add the controller and route for user sign up authentication
#### Description of Task to be completed?
Have the following endpoint working
POST /api/v1/auth/signup
#### How should this be manually tested?
After cloning the repo, cd into it, set the environment variables and run `npm run start:dev`
* Using postman, test for the above enpoint
* The body of the request should contain at least `firstname` `lastname` `email` `password`
#### What are the relevant pivotal tracker stories?
#166968465
#### Screenshots (if appropriate)
![signup](https://user-images.githubusercontent.com/30093122/60390144-d38e5280-9ac7-11e9-996f-d5c1e1de561f.png)
